docs/advanced/build_script.md

```diff
@@ -1,40 +1,40 @@
-# mbed SDK build script environment
+# mbed OS build script environment
 ## Introduction
-Mbed test framework allows users to test their mbed devices’ applications, build mbed SDK library, re-run tests, run mbed SDK regression, add new tests and get all this results automatically. Everything is done on your machine so you have a full control over compilation, and tests you run.
+mbed test framework allows users to test their mbed devices’ applications, build mbed OS library, rerun tests, run mbed OS regression, add new tests and receive all the results automatically. Everything occurs on your machine, so you have full control over compilation and the tests you run.
 
-It's is using Python 2.7 programming language to drive all tests so make sure Python 2.7 is installed on your system and included in your system PATH. To compile mbed SDK and tests you will need one or more supported compilers installed on your system.
+It uses Python 2.7 programming language to drive all tests, so make sure to install Python 2.7 and include it in your system PATH. To compile mbed OS and tests, you will need one or more supported compilers installed on your system.
 
-To follow this short introduction you should already:
-* Know what mbed SDK is in general.
-* Know how to install Python 2.7, ARM target cross compilers.
-* You have C/C++ programming experience and at least willingness to learn a bit about Python. 
+To follow this introduction, you should:
```

Comment:
A header of "requirements" or something like that will help - people often just scan the headers rather than read the test.